src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/SelectPart.cshtml

```diff
@@ -0,0 +1,26 @@
+@using OrchardCore.Forms.Models
+@model ShapeViewModel<SelectPart>
+@{
+    var formElementPart = Model.Value.ContentItem.As<FormElementPart>();
+    var formInputElementPart = Model.Value.ContentItem.As<FormInputElementPart>();
+    var elementId = formElementPart.Id;
+    var fieldName = formInputElementPart.Name;
+    var fieldId = !string.IsNullOrEmpty(elementId) ? elementId : !string.IsNullOrEmpty(fieldName) ? Html.GenerateIdFromName(fieldName) : default(string);
+    var fieldValue = Model.Value.DefaultValue;
+    var fieldClass = "form-control";
```

Comment:
why not just assign directly to html `class="form-control"` and have `fielderror` variable instead?

Reply:
Just keeping it the same as all the other form templates.
We could review them as a batch at any time

Reply:
OK